#test exception

int main() {
	return raise(1);
}

int raise(n:int) {
	if (n > 0) return 1/(n-n);
	while (1) {
		n += n;
	}
	return n;
}
